lisa/operating_system.py

```diff
@@ -409,7 +409,7 @@ class OpenBSD(BSD):
 
 
 class Fedora(Linux):
-    __fedora_release_pattern_version = re.compile(r"^.*release\s+([0-9\.]+).*$")
+    fedora_release_pattern_version = re.compile(r"^.*release\s+([0-9\.]+).*$")
```

Comment:
It looks don't need to be public.

Reply:
I can't use it from RedHat class if it is not public.

Reply:
Redhat inherits Fedora class, so should be accessible from there like this - super().__fedora_release_pattern_version

Reply:
I tried but didn't work, let me try again.

Reply:
`__` is private, cannot be used in subclasses, `_` can be.